rust/kcl-lib/src/std/planes.rs

```diff
@@ -25,7 +25,7 @@ pub async fn plane_of(exec_state: &mut ExecState, args: Args) -> Result<KclValue
         .map(|value| KclValue::Plane { value })
 }
 
-async fn inner_plane_of(
+pub async fn inner_plane_of(
```

Comment:
```suggestion
pub(crate) async fn inner_plane_of(
```
no need for this to be publicly usable outside the crate.